### ISSUE TITLE
[Logistic Circuit] Bias line may get missing when saving

### ISSUE THREAD

**Issue by YitaoLiang** [Juice-jl/ProbabilisticCircuits.jl#19]
as described in the titlle

**Comment by khosravipasha** [comment]
@YitaoLiang 
Yeah, it Juice, bias line is converted to a sum node in the root with one child that holds the biases, its a bit different than the python, version maybe this was the issue. 

Also the leafs don't have paramters, instead they are converted to sum nodes with both literals. And theta, 1-theta as params.
